### PR TITLE
Enhancement:save crypto wallet address 533

### PR DESCRIPTION
**Pull request by sana7044**
Now, we can  add, update and delete the crypto wallet address.

### CODE REVIEW

**Review by marclupanc — CHANGES_REQUESTED** [review]
There is an issue with removing the wallet. 
When connecting the wallet for the first time, we can see that we can access the "Add advertisement".
Then after removing it from profile page, and saving the profile page and when trying to create an advertisement we are still able to do that. 
Then when going back to the profile page (and refreshing) we can see that the wallet address is still linked to our profile. Please see the video

https://vimeo.com/1008435431?share=copy

File: src/components/Profile/ProfileTab.vue [hunk 1]
```diff
@@ -53,7 +53,7 @@
       <q-input class="non-selectable" debounce="400" label="Current Connected wallet Address" v-model.trim="currentWalletAddress" readonly>
         <Web3ModalComponent page_name="profile" />
 
-        <q-btn
+        <!-- <q-btn
```
[the remark this answers is not in the document]
Comment (sana7044) [reply src/components/Profile/ProfileTab.vue]: Yes, i removed the unused parts from this file.

File: src/components/Profile/ProfileTab.vue [hunk 2]
```diff
@@ -87,27 +111,50 @@
...
     <q-card>
       <q-card-section class="q-pb-none">
-        <h6 class="q-my-sm">set wallet address</h6>
+        <h6 class="q-my-sm">{{ isUpdate ?'Update':'Set' }} wallet address</h6>
       </q-card-section>
       <q-card-section>
         <span class="q-ml-sm">
-          Are you sure you want to set:
-          <b>{{ address }}</b>
+          Are you sure you want to {{ isUpdate ?'Update':'Set' }}:
```
[the remark this answers is not in the document]
Comment (sana7044) [reply src/components/Profile/ProfileTab.vue]: Yes, i changed this type.

**Comment by sana7044** [comment]
I have fixed all issues regrading the change requests.

**Review by marclupanc — APPROVED** [review]

**Pull request merged**